### DISCUSSION TITLE
When will you release a new version ?

### DISCUSSION THREAD

**Discussion by ageorge95** [Chia-Network/chia-blockchain#8134] [General]
As the title says, the latest version is from almost a month ago; but commits are made daily.

When will we get to enjoy these new commits via a new release ?

Thanks !

**Comment by ageorge95** [comment]
Helllllo !

No answers ?

**Comment by loppefaaret** [comment]
No, it is unlikely we release this week, but perhaps we get everything ready in next week - we avoid setting any dates

**Reply by ageorge95** [reply]
Thanks for your response 👍

**Comment by loppefaaret** [comment]
devs pulled the rug under me, see:
https://github.com/Chia-Network/chia-blockchain/releases for 1.2.4

**Comment by BrandtH22** [comment]
@ageorge95 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
